### ISSUE TITLE
AdminHomePageUiTest fails frequently at line 162

### ISSUE THREAD

**Issue by LowWeiLin** [TEAMMATES/repo#2961]
The failure is due to a true false mismatch.
![fail1](https://cloud.githubusercontent.com/assets/3910365/6078839/7fbfdeec-ae38-11e4-83dc-03effb8ef28c.png)

**Comment by ymoctavia** [comment]
I got another failure before line 162. 

@franklingu, did you change the feedbacksessionActions logic in #2941 ? This is causing adminHomePage fail now. Here is the screen shot:

![image](https://cloud.githubusercontent.com/assets/7552558/6080224/dcf5e44c-ae49-11e4-822c-61de92677c95.png)

The dropdown button is not disabled when it actually should be disabled.

**Comment by LowWeiLin** [comment]
I think they are separate issues, but yes, the master branch is failing there now. InstructorHomePageUiTest also seems to be affected by that

**Comment by franklingu** [comment]
yes I was changing that. but as @LowWeiLin said, I do not think that is the cause of this issue--all I did is to add disabled to links in dropdown box. previously the disabled was on the arrow, but apparent that  did not disable the dropdown

**Comment by LowWeiLin** [comment]
@franklingu 
The following tests/lines fail after #2848 was merged (separate issue should probably be created for this)
InstructorHomePageUiTest is failing at line 161
![image](https://cloud.githubusercontent.com/assets/3910365/6082545/339b01a2-ae5d-11e4-90cb-fa950a4ec583.png)

And AdminHomePageUiTest at line 148
![image](https://cloud.githubusercontent.com/assets/3910365/6082559/56036392-ae5d-11e4-8f41-51ba61e9fbe6.png)

**Comment by franklingu** [comment]
Okay. never see this test failure. create the issues and assign that to me :P
